visionos/ReactTestApp/AppDelegate.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
```

Comment:
Can we use symlinks to share code like we do in the `macos` folder?
```
% ls -Gl macos
total 8
drwxr-xr-x   3   96 13 jul  2022 Localizations/
drwxr-xr-x  11  352 29 jan 13:24 ReactTestApp/
drwxr-xr-x@  4  128 14 apr  2023 ReactTestApp.xcodeproj/
lrwxr-xr-x   1   19 13 jul  2022 ReactTestAppShared@ -> ../ios/ReactTestApp
drwxr-xr-x   4  128 13 jul  2022 ReactTestAppTests/
drwxr-xr-x   4  128 13 jul  2022 ReactTestAppUITests/
-rw-r--r--   1  131  8 sep 12:28 test_app.rb
```

Reply:
Ah, so that's how you did that. Sure.

Reply:
I think I went with copying the symlinked folder from `macos`, and the rest from `ios`. Thanks for the tip!